src/main/java/org/osiam/security/authorization/AccessTokenValidationService.java

```diff
@@ -143,9 +147,9 @@ private AccessToken validateAccessToken(String token) {
             LOGGER.warn(String.format("Unable to retrieve access token from auth-server %s: %s",
                     authServerHome, e.getMessage()));
             throw e;
-        } catch(OsiamRequestException e) {
+        } catch (UnauthorizedException e) {
             LOGGER.warn("Error validating access token", e);
```

Comment:
Even if it wasn't in this commti I think e should be consistent regarding the logging.